compiler/tests-common/tests/org/jetbrains/kotlin/codegen/ir/AbstractIrLineNumberTest.kt

```diff
@@ -24,10 +24,13 @@ abstract class AbstractIrLineNumberTest : AbstractLineNumberTest() {
 
     override fun compareCustom(psiFile: KtFile, wholeFile: File) {
         val fileText = psiFile.text
-        val expectedLineNumbers = normalize(
-            fileText.substring(fileText.indexOf("//") + 2)
-                .trim().split(" ").map { it.trim() }.toMutableList()
-        )
+        val expectedLineNumbers = fileText.split("\n".toRegex()).filter { line ->
```

Comment:
How about simplifying this to:

        val expectedLineNumbers = normalize(
            fileText.substring(Regex("// \\d+").find(fileText)!!.range.start + 2)
                .trim().split(" ").map { it.trim() }.toMutableList()
        )

Then we are looking for a line that starts with "// " with a number of digits after it. That should work to exclude the comment lines with text. That looks consistent with what the old backend expects from these tests.

Reply:
Looks reasonable since this is how old test is doing. I was a little bit overthinking this, was considering the case where there is a trailing comment line after line numbers

Reply:
I am also considering adding an optional "+" check before digit check, since there are line numbers start with a "+", although such case is not present in current test data yet.

Reply:
It wouldn't presented in future (answered in slack)